blocks/shared-styles/package.json

```
@@ -22,7 +22,10 @@
   },
   "peerDependencies": {
     "@wpmedia/engine-theme-sdk": "canary",
-    "@wpmedia/news-theme-css": "stable"
+    "@wpmedia/news-theme-css": "stable",
+    "@wpmedia/resizer-image-block": "canary",
+    "@wpmedia/placeholder-image-block": "canary",
```

```suggestion
    "@wpmedia/news-theme-css": "*",
    "@wpmedia/resizer-image-block": "*",
    "@wpmedia/placeholder-image-block": "*",
```

for future npm@7 usability